gui/src/components/s3/account-management.vue

```diff
@@ -853,6 +860,7 @@ export default class CortxAccountManagement extends Vue {
     };
   }
   public async mounted() {
+    this.checkPermissions();
```

Comment:
Please add await as checkPermissions is an async function

Reply:
await is added